src/nodes/chunk_append/planner.c

```diff
@@ -368,6 +368,9 @@ ts_chunk_append_get_scan_plan(Plan *plan)
 		case T_TidScan:
 		case T_ValuesScan:
 		case T_WorkTableScan:
+#if PG14_GE
+		case T_TidRangeScan:
+#endif
```

Comment:
There are similar checks for plan nodes inside `ca_append_begin` and `constraint_aware_append_plan_create`. Any reason to why you do not need to add this case there as well?

Reply:
Thanks for finding this. I have added TidRangeScan in these functions as well.